components/dashboard/Menu.tsx

```diff
@@ -152,6 +153,12 @@ export const getMenuItems = ({ intl, account, LoggedInUser }): MenuItem[] => {
       Icon: LayoutDashboard,
       if: !isAccountantOnly,
     },
+    {
+      section: ALL_SECTIONS.SEARCH,
+      Icon: Search,
+      label: intl.formatMessage({ id: 'Search', defaultMessage: 'Search' }),
+      if: LoggedInUser?.hasPreviewFeatureEnabled(PREVIEW_FEATURE_KEYS.SEARCH_RESULTS_PAGE),
```

Comment:
Can we rather reuse the existing "SEARCH_COMMAND" preview feature? To avoid having too many flags in there.